### PR TITLE
Format comments using roxygen2

### PR DESCRIPTION
**Pull request by marlycormar**
To test the `man` comments: 
```
?rarrior::get_latest_warrior_data_file_names
```

### CODE REVIEW

File: R/get_latest_warrior_data_file_names.R [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+#'
+#' @param from_directory A directory where to find the latest data.
+#' @return A list containing of key-value pairs
+#'         returned_list[['data']] to get the data file name,
+#'         returned_list[['metadata']] to get the metadata file name and,
+#'         returned_list[['date']] to get the date associated with the data.
+#' @examples
+#' get_latest_warrior_data_file_names()
+#' get_latest_warrior_data_file_names('/warrior_data_file_directory')
 get_latest_warrior_data_file_names <- function(from_directory='~/Downloads/') {
```
Comment (pbchase) [inline R/get_latest_warrior_data_file_names.R]: How about 

```
get_latest_warrior_data_file_names <- function(from_directory='~/Downloads/warrior_data') {
```

File: R/get_latest_warrior_data_file_names.R [hunk 2]
```diff
@@ -1,15 +1,16 @@
+#' Gets the latest downloaded files (in terms of date) from the provided
+#' from_directory, as well as the date reflected in the filenames.
+#' If no arguments are provided, the directory is assumed to be the '~/Downloads/' folder.
+#'
+#' @param from_directory A directory where to find the latest data.
+#' @return A list containing of key-value pairs
+#'         returned_list[['data']] to get the data file name,
+#'         returned_list[['metadata']] to get the metadata file name and,
+#'         returned_list[['date']] to get the date associated with the data.
+#' @examples
```
Comment (pbchase) [inline R/get_latest_warrior_data_file_names.R]: Add `@export` before the @example

File: DESCRIPTION [hunk 3]
```diff
@@ -9,3 +9,4 @@ Depends: R (>= 3.5.0)
 License: MIT
 Encoding: UTF-8
 LazyData: true
+RoxygenNote: 6.0.1
```
Comment (pbchase) [inline DESCRIPTION]: Add 

```
Imports:
  rdatil
```
after Depends

**Comment by marlycormar** [comment]
@pbchase Done!

**Pull request merged**